ObedZac/iREPORTER
# Ch implementing feedback 162369832

# 162369832 Implementing feedback 
### What does this PR do?
This PR implements some of the feedback I received from my learning facilitator
### Description of Task to be completed?
• Put badges on the Readme in the first line.
• Update Readme to have steps of reproducing the app in a new environment.
• Remove commented code and unused files from my repo.

### How should this be manually tested?
View the README.md to see if updates are as directed and also check through the repo to see that all unneeded files and code comments have been deleted.

### What are the relevant pivotal tracker stories?
162369832
